src/jit-analyze/jit-analyze.cs

```diff
@@ -643,18 +643,31 @@ class ScriptResolverPolicyWrapper : ICommandResolverPolicy
 
                 foreach (var line in rawLines)
                 {
-                    var fields = line.Split(new[] { ' ', '\t', '"' }, StringSplitOptions.RemoveEmptyEntries);
-                    if (fields.Length != 5)
+                    string manipulatedLine = line;
+
+                    int numFields = 5;
+
+                    Regex unixGitOutputRegex = new Regex(@"\{\w+\s=>\s\w+\}");
+                    if (unixGitOutputRegex.Matches(line).Count > 0)
+                    {
+                        string[] splitLine = unixGitOutputRegex.Split(line);
+                        manipulatedLine = String.Join("base", splitLine);
```

Comment:
I'm confused about what these two lines do. It would be helpful to write comments with an example, as well as adding an example for the case that is not in the "unix" format